client/src/components/graphs/OrganizationalChart.tsx

```diff
@@ -724,18 +735,21 @@ const CustomNode = ({
           />
         )}
       </div>
-      <LinkTo
-        modelType="Organization"
-        model={organization}
-        showAvatar={false}
-        showIcon={false}
-        style={{
-          minHeight: NODE_HEIGHT,
-          display: "flex",
-          padding: "5px 0px 5px 5px",
-          alignItems: "center"
-        }}
-      />
+      <CustomOrgLinkS>
+        <LinkTo
+          modelType="Organization"
+          model={organization}
+          showAvatar={false}
+          showIcon={false}
+          style={{
+            display: "-webkit-box",
+            WebkitLineClamp: 3,
+            WebkitBoxOrient: "vertical",
```

Comment:
These three are non-standard CSS (also `"-webkit-box"` is old and has since been replaced by Flexbox layouts).